README.md

```diff
@@ -1,22 +1,21 @@
 # Node Thread Pool :arrow_double_up: :on:
+
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 [![Dependabot](https://badgen.net/dependabot/dependabot/dependabot-core/?icon=dependabot)](https://badgen.net/dependabot/dependabot/dependabot-core/?icon=dependabot)
 [![npm w](https://img.shields.io/npm/dw/poolifier)](https://www.npmjs.com/package/poolifier)
 [![Actions Status](https://github.com/pioardi/node-pool/workflows/NodeCI/badge.svg)](https://github.com/pioardi/node-pool/actions)
 [![Coverage Status](https://coveralls.io/repos/github/pioardi/poolifier/badge.svg?branch=master)](https://coveralls.io/github/pioardi/poolifier?branch=master)[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
-[![NODEP](https://img.shields.io/static/v1?label=dependencies&message=no%20dependencies&color=brightgreen
-)](https://img.shields.io/static/v1?label=dependencies&message=no%20dependencies&color=brightgreen
-)
+[![NODEP](https://img.shields.io/static/v1?label=dependencies&message=no%20dependencies&color=brightgreen)](https://img.shields.io/static/v1?label=dependencies&message=no%20dependencies&color=brightgreen)
 [![Gitter](https://badges.gitter.im/poolifier/community.svg)](https://gitter.im/poolifier/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-<h2>Why Poolifier? </h2>
-Poolifier is used to perform heavy CPU bound tasks on nodejs servers, it implements thread pools ( yes, more thread pool implementations, so you can choose which one fit better for you ) using <a href="https://nodejs.org/api/worker_threads.html#worker_threads_worker_threads">worker-threads </a>.<br>
-With poolifier you can improve your <strong>performance</strong> and resolve problems related to the event loop.<br>
-Moreover you can execute your CPU tasks using an API designed to improve the <strong>developer experience</strong>.
+## Why Poolifier?
 
+Poolifier is used to perform heavy CPU bound tasks on nodejs servers, it implements thread pools (yes, more thread pool implementations, so you can choose which one fit better for you) using [worker-threads](https://nodejs.org/api/worker_threads.html#worker_threads_worker_threads).  
+With poolifier you can improve your **performance** and resolve problems related to the event loop.  
+Moreover you can execute your CPU tasks using an API designed to improve the **developer experience**.
 
+## Contents
```

Comment:
Compatibility and choose a pool links are no more working now

Reply:
Links should look like that: https://github.com/pioardi/poolifier/blob/clean-up-project/README.md#contents

Reply:
Could you tell me where the links are?

Reply:
> Could you tell me where the links are?

In the table of content: 

+ Choose a pool 
+ Team
+ ... 

I think you can also switch it to md syntax: 

```
##### Table of Contents  
[Headers](#headers)  
[Emphasis](#emphasis)  
...snip...    
<a name="headers"/>
## Headers
```

Reply:
Oh... thx, I didn't even noticed that there is a TOC ^^